<?php declare(strict_types = 1);

$ignoreErrors = [];
$ignoreErrors[] = [
    'rawMessage' => 'Method PhpCsFixer\\AbstractDoctrineAnnotationFixer@anonymous/src/Fixer/Internal/ConfigurableFixerTemplateFixer.php:520::configure() has PhpCsFixer\\ConfigurationException\\InvalidFixerConfigurationException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../src/Fixer/Internal/ConfigurableFixerTemplateFixer.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method PhpCsFixer\\StdinFileInfo::getATime() has RuntimeException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../src/StdinFileInfo.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method PhpCsFixer\\StdinFileInfo::getCTime() has RuntimeException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../src/StdinFileInfo.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method PhpCsFixer\\StdinFileInfo::getGroup() has RuntimeException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../src/StdinFileInfo.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method PhpCsFixer\\StdinFileInfo::getInode() has RuntimeException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../src/StdinFileInfo.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method PhpCsFixer\\StdinFileInfo::getLinkTarget() has RuntimeException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../src/StdinFileInfo.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method PhpCsFixer\\StdinFileInfo::getOwner() has RuntimeException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../src/StdinFileInfo.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method PhpCsFixer\\StdinFileInfo::getSize() has RuntimeException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../src/StdinFileInfo.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method PhpCsFixer\\StdinFileInfo::getType() has RuntimeException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../src/StdinFileInfo.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method PhpCsFixer\\Fixer\\ConfigurableFixerInterface@anonymous/tests/Console/Command/DescribeCommandTest.php:513::configure() has PhpCsFixer\\ConfigurationException\\InvalidFixerConfigurationException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../tests/Console/Command/DescribeCommandTest.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method Symfony\\Component\\Filesystem\\Filesystem@anonymous/tests/Console/Command/DocumentationCommandTest.php:65::dumpFile() has Symfony\\Component\\Filesystem\\Exception\\IOException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../tests/Console/Command/DocumentationCommandTest.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method PhpCsFixer\\Fixer\\ConfigurableFixerInterface@anonymous/tests/FixerFactoryTest.php:445::configure() has PhpCsFixer\\ConfigurationException\\InvalidFixerConfigurationException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../tests/FixerFactoryTest.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method PhpCsFixer\\Fixer\\ConfigurableFixerInterface@anonymous/tests/FixerFactoryTest.php:521::configure() has PhpCsFixer\\ConfigurationException\\InvalidFixerConfigurationException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../tests/FixerFactoryTest.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method Symfony\\Component\\Process\\Process@anonymous/tests/Linter/ProcessLintingResultTest.php:33::wait() has Symfony\\Component\\Process\\Exception\\LogicException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../tests/Linter/ProcessLintingResultTest.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method Symfony\\Component\\Process\\Process@anonymous/tests/Linter/ProcessLintingResultTest.php:33::wait() has Symfony\\Component\\Process\\Exception\\ProcessSignaledException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../tests/Linter/ProcessLintingResultTest.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method Symfony\\Component\\Process\\Process@anonymous/tests/Linter/ProcessLintingResultTest.php:33::wait() has Symfony\\Component\\Process\\Exception\\ProcessTimedOutException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../tests/Linter/ProcessLintingResultTest.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method Symfony\\Component\\Process\\Process@anonymous/tests/Linter/ProcessLintingResultTest.php:53::getErrorOutput() has Symfony\\Component\\Process\\Exception\\LogicException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../tests/Linter/ProcessLintingResultTest.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method Symfony\\Component\\Process\\Process@anonymous/tests/Linter/ProcessLintingResultTest.php:53::wait() has Symfony\\Component\\Process\\Exception\\LogicException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../tests/Linter/ProcessLintingResultTest.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method Symfony\\Component\\Process\\Process@anonymous/tests/Linter/ProcessLintingResultTest.php:53::wait() has Symfony\\Component\\Process\\Exception\\ProcessSignaledException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../tests/Linter/ProcessLintingResultTest.php',
];
$ignoreErrors[] = [
    'rawMessage' => 'Method Symfony\\Component\\Process\\Process@anonymous/tests/Linter/ProcessLintingResultTest.php:53::wait() has Symfony\\Component\\Process\\Exception\\ProcessTimedOutException in PHPDoc @throws tag but it\'s not thrown.',
    'count' => 1,
    'path' => __DIR__ . '/../../../tests/Linter/ProcessLintingResultTest.php',
];

return ['parameters' => ['ignoreErrors' => $ignoreErrors]];
